src/crafting_gui.h

```diff
@@ -2,9 +2,12 @@
 #ifndef CRAFTING_GUI_H
 #define CRAFTING_GUI_H
 
+#include "point.h"
```

Comment:
```suggestion
struct tripoint;
```
No need to include the file here in the header. A forward declaration is enough. If necessary, you can include the file in the cpp file itself.